website/project/model.py

```diff
@@ -3703,26 +3779,25 @@ def find_question(self, qid):
                 if question_id == qid:
                     return question
 
-    # TODO: uncomment when we support commenting
-    # def get_comments(self):
-    #    """ Returns a list of all comments made on a draft in the format of :
-    #    [{
-    #      [QUESTION_ID]: {
-    #        'question': [QUESTION],
-    #        'comments': [LIST_OF_COMMENTS]
-    #        }
-    #    },]
-    #    """
-    #
-    #    all_comments = []
-    #    for question_id, value in self.registration_metadata.iteritems():
-    #        all_comments.append({
-    #            question_id: {
-    #                'question': self.find_question(question_id),
-    #                'comments': value['comments'] if 'comments' in value else ''
-    #            }
-    #        })
-    #    return all_comments
+    def get_comments(self):
+        """ Returns a list of all comments made on a draft in the format of :
+        [{
+          [QUESTION_ID]: {
```

Comment:
I think he is referring to `QUESTION_ID` being in square brackets which I think is a mistake and should probably be in single quotes